library/item/enchantment/src/main/java/org/quiltmc/qsl/enchantment/api/EnchantmentEvents.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2023 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.enchantment.api;
+
+import java.util.List;
+
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.Nullable;
+
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.enchantment.EnchantmentHelper;
+import net.minecraft.enchantment.EnchantmentLevelEntry;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.screen.AnvilScreenHandler;
+
+import org.quiltmc.qsl.base.api.event.Event;
+import org.quiltmc.qsl.enchantment.api.context.EnchantingContext;
+
+public final class EnchantmentEvents {
+	/**
+	 * An event that is called after random enchantments are generated, but before they are selected to be applied on an item.
```

Comment:
Maybe it's a me thing, but it took me way too long to guess what "random enchantments" were for
```suggestion
	 * An event that is called after random enchantments are generated (e.g. for an enchantment table), but before they are selected to be applied on an item.
```